llvm/test/tools/llvm-objcopy/tool-options.test

```diff
@@ -1,4 +1,6 @@
 ## An error must be reported if a required argument value is missing.
-# RUN: not llvm-objcopy --only-section 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE %s
-# RUN: not llvm-objcopy -O 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE %s
-# CHECK-NO-VALUE: error: argument to '{{.*}}' is missing
+# RUN: not llvm-objcopy --only-section 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE-ONLY-SECTION %s
+# CHECK-NO-VALUE-ONLY-SECTION: error: argument to '--only-section' is missing
```

Comment:
You're still not checking the latter part of the message, and in particular the substitution for the argument count.

Reply:
I added the check for the full message in 49c078a. Thanks.